api/app/utils/review.inference.ts

```diff
@@ -59,6 +59,16 @@ const buildIsExpiredAndNotArchivedCondition = (currentTime: number) => {
   };
 };
 
+const buildMergedCondition = () => {
+  const mergedQuery = {
+    [Op.and]: [ //ensuring all conditions are met
+      { mergedAt: { [Op.not]: null } }, // has been merged
+      { archivedAt: { [Op.not]: null } }, // has been archived
+    ]
```

Comment:
This means that you considered "merged" a review that has a `mergedAt` AND a `archiveAt` field.
This is wrong and you can verify that by looking at the reviews table in our database.

We are also getting merged reviews with similar logic here:
https://github.com/bitcointranscripts/transcription-review-backend/blob/fef777688f2f02b4dead7dfae93da8a5b3217306/api/app/controllers/review.controller.ts#L316-L318

Reply:
That is correct, thought archivedAt should is necessary for a complete review cycle.

Reply:
> That is correct, thought archivedAt should is necessary for a complete review cycle.

I don't understand what you mean with this. Can you elaborate?

Reply:
I meant I was thinking for a merged condition, archivedAt has to be populated in the DB